src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,71 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+    public static final String DELIMITER = ",";
+    public static final String OPERATION = "buy";
+    private int supplyAmount = 0;
+    private int buyAmount = 0;
```

Comment:
These class fields should be local variables in the `readDataFromTheFile` method since they are used to accumulate values during the reading process and do not represent the state of the class object.